_includes/head.html

```diff
@@ -31,4 +31,5 @@
   </style>
 <![endif]-->
 
-<meta http-equiv="cleartype" content="on">
\ No newline at end of file
+<meta http-equiv="cleartype" content="on">
+<script async defer src="{{ '/assets/js/main.min.js' | absolute_url }}"></script>
```

Comment:
Does adding both `async` and `defer` attributes actually work? I was under the impression you had 3 options with scripts.

> There are three possible modes that can be selected using these attributes. If the async attribute is present, then the script will be executed asynchronously, as soon as it is available. If the async attribute is not present but the defer attribute is present, then the script is executed when the page has finished parsing. If neither attribute is present, then the script is fetched and executed immediately, before the user agent continues parsing the page.

Source: https://www.w3.org/TR/html5/scripting-1.html#attr-script-async